### ISSUE TITLE
Move Web IDE logo somewhere because it looks like a change-view button

### ISSUE THREAD

**Issue by anton-trunov** [tact-lang/web-ide#183]
<img width="73" alt="image" src="https://github.com/user-attachments/assets/f94f3d64-aa62-4125-bad0-e3461103bd01" />

Often times when I need to switch to the project view, I accidentally hit the logo. It is placed in the vertical column with some functional buttons but the logo is not a button, so this is confusing

**Comment by rahulyadav-57** [comment]
Yes, it's a bit confusing. We can add a separator line and disable the logo's clickability to make it clearer.

<img width="322" alt="image" src="https://github.com/user-attachments/assets/99b16ef1-c3d4-444a-9cbe-44fcd32a9b46" />

**Comment by anton-trunov** [comment]
Or move it somewhere else from the left-side bar

**Comment by novusnota** [comment]
Another idea is to make it smaller and place next to the name of panes, like "<logo> Explorer", "<logo> Build & Deploy", etc.

**Comment by rahulyadav-57** [comment]
@novusnota I think this would be a better. We will have space for more options in side-menu.
